pinot-spi/src/main/java/org/apache/pinot/spi/utils/BytesUtils.java

```diff
@@ -40,6 +41,25 @@ public static byte[] toBytes(String stringValue) {
     }
   }
 
+  /**
+   * Convert a string to UTF8 byte array.
+   * @param stringValue
+   * @return UTF8 byte array.
+   */
+  public static byte[] toUTF8Bytes(String stringValue) {
```

Comment:
Are these 2 methods used? IIRC we used to have these methods in `StringUtil`, but then removed them because there is no much point adding a util function for a one code.

Reply:
Used only by test. Moved to test class.